YaseminAydin.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+	<head>
+		<title> This is Me </title>
+		 <a> https://www.linkedin.com/in/yasemin-aydin-1414959b/ </a>
+	</head>
+	<body>
+		<h1>Firs Years of My Life</h1>
+		<p>
+		Hello,
+		My name is Yasemin Aydin and I was born in Turkey/Malatya. When I was aroun 5 years we moved to Bursa which is more western city than Malatya. I comleted my education in Bursa till University.
+		
+		</p>
+		<br>
+		<p>
+		<h1> New Country New Dreams </h1>
+		<p>
+		When I decided to study at the univesity the life opened me a new country's door til the end, it was the biggest mlestone in my life. I found my love there and get marriedwith him. 
+		Also, I expand overseas with Russia where I studied my university, but it was only the first, after living in Russia for six years I was sure that I will live in different parts of the worl till the end of my life like a traveller.
+		Ultimately, Canada can make a dot to my dream, because I feel like I found my soul country to live in. Here, I make a lot of efford to structure a new life from 0. I know this nt east but I always belive myself and again I say the same words I CAN DO IT.<br>
+		<img src="https://www.google.com/imgres?imgurl=https%3A%2F%2Fwww.globalgrasshopper.com%2Fwp-content%2Fuploads%2F2017%2F01%2Fmoscow-web2.jpg&imgrefurl=https%3A%2F%2Fglobalgrasshopper.com%2Fdestinations%2Feurope%2F10-of-the-most-beautiful-places-to-visit-in-russia%2F&docid=ZQoFOm9Q0Jfe9M&tbnid=e0ln31yc1lqXUM%3A&vet=10ahUKEwjfn7f62tjjAhVVLs0KHXKLA48QMwh5KAIwAg..i&w=1024&h=654&bih=608&biw=1366&q=russia&ved=0ahUKEwjfn7f62tjjAhVVLs0KHXKLA48QMwh5KAIwAg&iact=mrc&uact=8" alt= "My First Country" width="100"><br>
```

Comment:
Your images aren't working. You should save the images you want to use to your computer (right click and select "save image as"). Create a new folder called `images` and put your images in there.

Then you can include your images like this:
`<img src="images/yourfilename.jpg">`
Replace `yourfilename.jpg` with the file name of your image.